### ISSUE TITLE
`GKS not in proper state` error with `GR_jll` and `GR` v0.57 on Windows

### ISSUE THREAD

**Issue by EHBaozi** [jheinen/GR.jl#383]
To produce the error, simply `using GR` and `plot([1, 2, 3])`. On my machine it reproduces consistently.

There are similar issues already but most of them are regarding Linux. This error occurs only with `GR_jll` provided binaries which is now the default on v0.57, however the same error also occurred with v0.55 (and I believe v0.54 and some other earlier versions as well) when I manually set the GR provider to be BinaryBuilder.

Currently the workarounds:

1. Manually set `ENV["JULIA_GR_PROVIDER"] = "GR"` and rebuild GR.
2. Set `ENV["GKSWSTYPE"] = "41"`, which works but results in much lower quality plots (at least interactively; haven't checked plot saving).
3. When imported, GR sets a `"GKS_QT"` environment variable that looks like `set PATH=...... & C:\path\to\artifact\gksqt.exe`. If I invoke `gksqt.exe` using exactly this command in a separate cmd shell after `using GR` then plotting works just fine.

`versioninfo()`:
```
Julia Version 1.6.0
Commit f9720dc2eb (2021-03-24 12:55 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, skylake)
Environment:
  JULIA_NUM_THREADS = 4
```

`Pkg.status()`:
```
      Status `C:\Users\username\.julia\environments\v1.6\Project.toml`
  [537997a7] AbstractPlotting v0.16.6
  [6e4b80f9] BenchmarkTools v0.7.0
  [336ed68f] CSV v0.8.4
  [159f3aea] Cairo v1.0.5
  [13f3f980] CairoMakie v0.4.2
  [a8cc5b0e] Crayons v4.0.4
  [a93c6f00] DataFrames v0.22.7
  [864edb3b] DataStructures v0.18.9
  [31c24e10] Distributions v0.24.15
  [7a1cc6ca] FFTW v1.3.2
  [e9467ef8] GLMakie v0.2.4
  [28b8d3ca] GR v0.57.0
  [c91e804a] Gadfly v1.3.2
  [7073ff75] IJulia v1.23.2
  [a98d9a8b] Interpolations v0.13.1
  [c8e1da08] IterTools v1.3.0
  [b964fa9f] LaTeXStrings v1.2.1
  [23fbe1c1] Latexify v0.14.12
  [2fda8390] LsqFit v0.12.0
  [eff96d63] Measurements v2.5.0
  [961ee093] ModelingToolkit v5.14.0
  [0987c9cc] MonteCarloMeasurements v0.10.1
  [872cf16e] NaturallyUnitful v0.3.0
  [5fb14364] OhMyREPL v0.5.10
  [429524aa] Optim v1.3.0
  [8314cec4] PGFPlotsX v1.2.10
  [5ad8b20f] PhysicalConstants v0.2.1
  [f0f68f2c] PlotlyJS v0.14.1
  [91a5bcdd] Plots v1.11.2
  [c3e4b0f8] Pluto v0.12.21
  [7f904dfe] PlutoUI v0.7.1
  [438e738f] PyCall v1.92.2
  [d330b81b] PyPlot v2.9.0
  [1fd47b50] QuadGK v2.4.1
  [6f49c342] RCall v0.13.10
  [ce6b1742] RDatasets v0.7.4
  [295af30f] Revise v3.1.14
  [f2b01f46] Roots v1.0.8
  [276daf66] SpecialFunctions v1.3.0
  [f3b207a7] StatsPlots v0.14.19
  [0c5d862f] Symbolics v0.1.12
  [028f657a] TotalLeastSquares v1.7.0
  [bc48ee85] Tullio v0.2.13
  [1cfade01] UnicodeFun v0.4.1
  [1986cc42] Unitful v1.6.0
  [a7773ee8] UnitfulAtomic v1.0.0
  [45397f5d] UnitfulLatexify v1.4.0
  [42071c24] UnitfulRecipes v1.1.1
  [0f1e0344] WebIO v0.8.15
  [e88e6eb3] Zygote v0.6.8
  [c4c688b2] jlpkg v1.3.2
```

**Comment by jheinen** [comment]
Unfortunately, I can only confirm that I can reproduce this behavior. For whatever reason, you can no longer start the `gksqt.exe` program from Julia, but only outside the REPL. This is of course suboptimal. Whether it is related to changes in Julia 1.6, the BinaryBuilder or even any Windows security settings - I can't say at the moment.

Therefore, method 1 (manually set `ENV["JULIA_GR_PROVIDER"] = "GR"` and rebuild GR) is currently the only workaround.

I was concerned from the beginning that this extremely long search path in `GR_jll` would work under Windows.

**Comment by EHBaozi** [comment]
Previously on Julia v1.5.4 and GR v0.55 I had the same behavior, so probably not related to Julia 1.6.

**Comment by jheinen** [comment]
I slightly modified the login in GR#master:

```
ENV["GKS_QT"] = ""
using GR
plot(randn(10))
```

... works, but when you close the `GKS QtTerm` windows, it is not automatically restarted with the next plot command. So it's just another workaround - but it shows, that there is a problem with this huge path within GR_jll.

**Comment by jheinen** [comment]
The problem should be fixed with version [0.57.1](https://github.com/sciapp/gr/releases/tag/v0.57.1
).

**Comment by EHBaozi** [comment]
Can confirm. Thank you!
